<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="cb_address"/>
        </item>
        <item>
         <widget class="QToolButton" name="btn_run">
          <property name="text">
           <string>run</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVideoWidget" name="video_widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btn_run</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
